packages/server/graphql/mutations/startCheckIn.ts

```diff
@@ -38,6 +39,14 @@ export default {
     if (!isTeamMember(authToken, teamId)) {
       return standardError(new Error('Team not found'), {userId: viewerId})
     }
+    const teams = await getTeamsByIds([teamId])
+    const team = teams[0]!
+    if (!isPaid(team)) {
+      const errMsg = isLocked(team)
+        ? "Wow, you're determined to use Parabol! That's awesome! Do you want to keep sneaking over the gate, or walk through the door with our Sales team?"
+        : 'Team is not paid'
```

Comment:
+1 I think we could provide a little more info here. Perhaps instead of `Team is not paid` we could say something like `Unable to start a meeting as your team has an overdue payment`?

Reply:
great idea, updated it to: 'Sorry! We are unable to start your meeting because your team has an overdue payment'